db/migrate/20231107142971_add_census_type_data_to_decicid_vocdoni_elections.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class AddCensusTypeDataToDecicidVocdoniElections < ActiveRecord::Migration[6.1]
+  def change
+    add_column :decidim_vocdoni_elections, :internal_census, :boolean, default: false
+    add_column :decidim_vocdoni_elections, :verification_types, :string, array: true, default: []
+    add_column :decidim_vocdoni_elections, :census_attributes, :jsonb, default: {}
```

Comment:
Let's do all the changes in the same migration 
```suggestion
    add_column :decidim_vocdoni_elections, :census_attributes, :jsonb, default: {}
    add_column :decidim_vocdoni_elections, :last_census_update_records_added, :integer
    add_column :decidim_vocdoni_elections, :census_last_updated_at, :datetime
    add_column :decidim_vocdoni_voters, :in_vocdoni_census, :boolean, default: false
```